mittab/templates/base/_navigation.html

```diff
@@ -71,6 +72,7 @@
             <a class="dropdown-item {%active request view_backups %}" href="{{ view_backups }}">View Backups</a>
             <a class="dropdown-item {%active request backup %}" href="{{ backup }}">Backup Current</a>
             <a class="dropdown-item {%active request tab_cards %}" href="{{ tab_cards }}">View Tab Cards</a>
+            <a class="dropdown-item {%active request download_archive %}" href="{{ download_archive }}">Create DebateXML</a>
```

Comment:
Can you add alt help here explaining what this is? Something like "Debate XML is a standardized format used across multiple debate events"

Reply:
Done. Wondering why there's a separate `navigation.html` with the same links. Used anywhere?

Reply:
No, I probably just forgot to delete that when I updated the UI. You can leave it as-is